app/templates/editions/polaris.hbs

```diff
@@ -11,6 +11,21 @@
     a new <a href="https://github.com/embroider-build/embroider">build pipeline</a> that hooks into the tool of your choice (webpack, rollup, etc), and a new <a href="https://github.com/NullVoxPopuli/ember-resources">reactivity system</a>.
   </p>
 
+  <h2>The pillars of Polaris</h2>
+  <p>The polaris edition of Ember is built on top of a selection of core concepts.</p>
+
+  <h3>1. First-class TypeScript Support</h3>
+  <p>Ember has embraced TypeScript as a first-class citizen next to JavaScript. In practice this means you can build apps using TypeScript without having to worry about incompatibility with external type definitions of Ember itself. As of version 5.1, Ember ships with built-in stable types. Read all about it and how to get started <a href="https://blog.emberjs.com/stable-typescript-types-in-ember-5-1">in the announcement post.</a> Glint, the TypeScript-powered tooling for Glimmer templates, also shipped as a stable release after two years of public preview. For more information, read the <a href="https://blog.emberjs.com/glint-1-0-released">release blog post</a>.</p>
+
+  <h3>2. Template tag components</h3>
+  <p>The template tag syntax is the new authoring format for components in Ember. It addresses some long-standing issues with the current format: explicit imports prevent naming conflicts, the new ability to introduce locally-scoped code, improved interoperability with third-party JavaScript libraries in the template, and aligning test syntax with app syntax. It also unlocks the road to new capabilities such as code splitting. See the <a href="https://github.com/ember-template-imports/ember-template-imports">ember-template-imports</a> repository to get started today!</p>
```

Comment:
```suggestion
  <p>The template tag syntax is the new authoring format for components in Ember. It addresses some long-standing developer experience challenges with the current format, and incorporates community feedback. Improvements include: explicit imports prevent naming conflicts, the new ability to introduce locally-scoped code, improved interoperability with third-party JavaScript libraries in the template, and aligning test syntax with app syntax. It also helps unlock the road to new capabilities such as code splitting in the future. See the <a href="https://github.com/ember-template-imports/ember-template-imports">ember-template-imports repository </a> to get started today!</p>
```

Optional: highlight the role of community feedback

I have a question about code splitting. I softened the language about it in the last sentence here. I’m assuming that code splitting is not yet available, and this is more like a stepping stone. Is that correct?